data/human/hails.txt

```diff
@@ -2082,6 +2082,98 @@ phrase "friendly civilian"
 		"Zug. I used to watch the militia ships fight against the pirate ships in the sky from my porch."
 		"New Holland. It gets too boring there, though, so I fly the stars instead."
 		"Bounty. I bought my first ship when the manufacturing industry started to develop there."
+		
+phrase "pyramid_scheme_company"
+	word
+		"Magellan "
+		"Dehydrated "
+		"Pontificated "
+		"General "
+		"Sacrosanct "
+		"Liberated "
+		"Literal "
+		"Family "
+		"Hyper-real "
+	word
+		"Dynamix"
+		"Unlimited"
+		"Orbital"
+		"Gigalight"
+		"Traders"
+		"Voidcraft"
+		"Whispers"
+
+phrase "friendly civilian"
+	word
+		"Hi! Good to see you, uh, "
+		"Well, if it isn't "
+		"Hi there, uhm, "
+		"I've been trying to get a hold of you, "
+	word
+		"Captain <Last>. "
+	word
+		"I'm living the dream as a CEO of my own company. "
+		"I used to be just like you, but now I'm really somebody! "
+		"Don't you want to be your own boss? "
+		"Everybody is talking about this slick new fad! "
+	word
+		"You could get rich selling for me!"
+		"Get in on the ground floor!"
+		"We're about to go orbital, you should sign up while you still can!"
+	word
+		" "
+	word
+		" "
+	word
+		" "
+	word
+		"Find me "
+		"You can get me "
+		"Call me "
+		"Drop me a line "
+		"Buzz me "
+	word
+		"at ${pyramid_scheme_company}. "
+	word
+		"Ask for ${name}."
+		"Tell them ${name} sent you."
+		"Mention my name, ${name}, for a discount on your first order!"
+
+phrase "friendly civilian"
+	word
+		"All I want is one day without "
+		"I just bought this ship and it already has "
+		"The warranty for my ship expired a week ago, and wouldn't you know it, I just found "
+	word
+		"a fuel leak! "
+		"a loose coupling! "
+		"a fault in my nav-array! "
+		"a cascading failure in the power supply! "
+		"a severe oxygen leak! "
+		"an engine overheating! "
+		"unsynchronized repulsors! "
+		"a parasitic shield disharge! "
+		"a leaky fuel cell! "
+		"unsynchronized repulsors! "
+		"a parasitic shield disharge! "
+		"a leaky fuel cell! "
+	word
+		"The maintenance on this old bucket will cripple me."
+		"It's just one thing after another on these old ships."
+		"I wouldn't complain, but today is my birthday!"
+		"And it's not as though parts are cheap these days."
+		"It could be worse, I suppose. I could fly into a black hole."
+		"It's not like the customers even care, they just want their cargo yesterday."
+		"What else can you expect from shoddy Syndicated Systems manufacturing?"
```

Comment:
The ship that the hailing merchant has won't necessarily be a Syndicate one.

Reply:
Not necessarily, but most ships will have some sort of Syndicate subsystems. I'm thinking doors, oxygen, life support, hyperdrives, cybernetics/ship controls, data panels, etc. That's why the Free Worlds needs the Syndicate for their ship manufacturing.
From Hephaestus:
> Syndicated Systems, which manufactures parts used in shipyards throughout human space